src/main.rb

```diff
@@ -4,26 +4,24 @@
 Bundler.setup(:default)
 require 'ruby-scanner-scaffolding'
 require 'ruby-scanner-scaffolding/healthcheck'
-require_relative "./ssh_worker"
+require_relative './ssh_worker'
 
-set :port, 8080
+set :port, 8_080
```

Comment:
Why that?

Reply:
Thats the recommended way to format numbers larger numbers in ruby.
`_` is the thousands separator.

Reply:
ok 👍